requirements.txt

```diff
@@ -15,6 +15,7 @@ decorator==4.4.2
 descartes==1.1.0
 dictdiffer==0.8.1
 docutils==0.15.2
+Fiona==1.8.20
```

Comment:
This is really likely to be something specific to my own machine, but I get fatal errors out of Fiona unless I pin to this specific version. I'm planning on getting a new machine soon, so hopefully this is just temporary, and I'm happy to remove this and keep the pinning as a local thing on my machine only if anybody objects to it being here.

Reply:
i have a vague memory that we deleted this from requirements as it was failing with windows users; geopandas handles fiona install, maybe if we're adding this, we should put it after geopandas on the list? (I think it installs top to bottom... it wont be alphabetical anymore though 😱 )

Reply:
If it's tricky, and it only really help me, I think maybe I will just remove it and manage that locally on my machine. I low key hate these libraries with native dependencies 😆